### ISSUE TITLE
SOSA: SampleCollection

### ISSUE THREAD

**Issue by KathiSchleidt** [w3c/sdw#1417]
Parallel to the `ObservationCollection`, a `SampleCollection` would be valuable.

**Comment by sgrellet** [comment]
Interesting, how would you define such a collection ? 

For example in the field of Water Quality, if you take a bucket of water from the river, bucket which is sub-sampled.
Would the sum of the sub-samples be a SampleCollection for you ?

Or would you make it more flexible in a 'whatever grouping of Samples deemed relevant' approach ?

**Comment by dr-shorthair** [comment]
I think the expectation would be that, like an `ObservationCollection`, the members of a `SampleCollection` share some common property(s). 

For example, all the members of a collection 
- share the same value for their `isSampleOf` property (i.e. were the result of an act-of-Sampling with the same `feature of interest`), or 
- were the result of an act-of-Sampling using the same `procedure`. 
- were the result of an act-of-Sampling by the same `Sampler`. 

etc

**Comment by KathiSchleidt** [comment]
Exactly. Parallel to the ObservationCharacteristics defined in OMS, that we use to describe the content of the ObservationCollection, I believe we'd need SamplingCharacteristics to describe the SampleCollection.

**Comment by sgrellet** [comment]
ok. We'll then need which Characteristics we are speaking about: SampleCharacteristics , SamplingCharacteristics  etc...

**Comment by KathiSchleidt** [comment]
I'd assumed `SamplingCharacteristics`, as `Sampling` provides all relevant information on how the `Sample` was created, thus best to guide the `Collection`

**Comment by KathiSchleidt** [comment]
`SampleCollection` and `ObservationCollection` can share the same superclass, but the ObservationCharacteristics or SamplingCharacteristics that describe the nature of the collection will be quite different.

Further question is if we keep the approach from OMS that allows for 2 different types of collections:

- Homogeneous: all `Observations` contained are of a similar nature; if a property (NOT just `ObservableProperty`, in this meaning, this could also be the `Procedure`, `Observer`, `Host`...) is provided, this property applies to all `Observations` contained in the `Collection`
- Summarizing: a wider grab-bag type of collection; if a property is provided, there is an `Observation` in the `Collection` with this property

A further question is if we keep the structure of 2 classes (ObservationCollection and ObservationCharacteristics), or if we lump this into one class

**Comment by sgrellet** [comment]
> "but the ObservationCharacteristics or SamplingCharacteristics that describe the nature of the collection will be quite different."

+1

- Homogeneous VS Summarizing :
 
 I feel we need to create the concept of Sample collection because semantically we have that need. but going down to homogeneous VS summarizing, I'm not sure, is this a level of contraint we want to have in SOSA ?  I feel it's almost close to a 'serialization optimisation' level (to avoid repeating the same values when shared by all collection members)
